tests/test_obj_store.py

```diff
@@ -83,8 +83,7 @@ def test_cancel_run(cpu_cluster):
     print_fn = rh.function(fn=do_printing_and_logging, system=cpu_cluster)
     key = print_fn.remote()
     assert isinstance(key, str)
-    res = cpu_cluster.cancel(key)
-    assert res == "Cancelled"
+    cpu_cluster.cancel(key)
     try:
```

Comment:
this test will still pass here even if cancel doesn't successfully do what it was meant to do

Reply:
Great point, and turned out it wasn't cancelling correctly! Fixed